pr_testing/run-pr-comparisons

```diff
@@ -387,21 +387,22 @@ if [ "X$RUN_DEFAULT_COMP" = Xtrue ]; then
 fi
 echo "Jobs: $(jobs -p | wc -l)"
 jobs
-
 # --------------------------------------------------------------------------
 # maxmem-profile comparison
 # --------------------------------------------------------------------------
+echo "Started maxmem-profile comparison at `date`"
 OUTPUT_DIR=$WORKSPACE/results/maxmem-comparison
 #create the output dir
 mkdir -p $OUTPUT_DIR
 for WF in ${WORKFLOWS_TO_COMPARE//,/ }; do
```

Comment:
@gartung , `$WORKFLOWS_LIST` jenkins env variable contaions comma separated list of all workflows.
`$WORKFLOWS_TO_COMPARE` contains the workflows which ran successfully for both IB and PRs (https://github.com/cms-sw/cms-bot/blob/master/pr_testing/run-pr-comparisons#L211-L227). So some time if there are relvals failure in IBs then you might miss them for maxmem_profile too.